extensions.toml

```
@@ -1895,3 +1895,7 @@ version = "0.3.3"
 [ziggy]
 submodule = "extensions/ziggy"
 version = "0.0.1"
+
+[zed-symbols]
+submodule = "extensions/zed-symbols
```

```suggestion
submodule = "extensions/zed-symbols"
```
You're missing a `"` here

---

thanks for observing well